### PR TITLE
Set connection to local mongodb, try to get data in the collection

### PR DESCRIPTION
**Pull request by thuyvy3110**
Connect to local database to get sample of blog Post portfolio data

### CODE REVIEW

File: src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.controller;
+
+public class CommentController {
```
Comment (DeKal) [inline src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java]: this is just a suggestion, we dont need to add a Comment right now :d

File: src/main/java/com/dawn/jat/illuminati/comment/entity/CommentEntity.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.entity;
+
+public class CommentEntity {
```
Comment (DeKal) [inline src/main/java/com/dawn/jat/illuminati/comment/entity/CommentEntity.java]: this is just a suggestion, we dont need to add a Comment right now :d

File: src/main/java/com/dawn/jat/illuminati/comment/repository/CommentRepository.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.repository;
+
```
Comment (DeKal) [inline src/main/java/com/dawn/jat/illuminati/comment/repository/CommentRepository.java]: this is just a suggestion, we dont need to add a Comment right now :d

File: src/main/java/com/dawn/jat/illuminati/comment/service/CommentService.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.service;
```
Comment (DeKal) [inline src/main/java/com/dawn/jat/illuminati/comment/service/CommentService.java]: this is just a suggestion, we dont need to add a Comment right now :d

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -1 +1,6 @@
+app.name=Illuminati on Local
+app.description=${app.name}
+spring.profiles.active=${SPRING_ACTIVE_PROFILE:"dev"}
```
Comment (DeKal) [inline src/main/resources/application.properties]: spring.profiles.active=${SPRING_ACTIVE_PROFILE:""} to set it to default

File: src/main/resources/application-staging.properties [hunk 6]
```diff
@@ -0,0 +1,2 @@
+app.name=Illuminati on Staging
+app.description=${app.name}
```
Comment (DeKal) [inline src/main/resources/application-staging.properties]: we need to set 
```
spring.data.mongodb.uri=${ENV_VAR}
spring.data.mongodb.database=${ENV_VAR}
```

File: src/main/resources/application-prod.properties [hunk 7]
```diff
@@ -0,0 +1,2 @@
+app.name=Illuminati on Production
+app.description=${app.name}
```
Comment (DeKal) [inline src/main/resources/application-prod.properties]: we need to set 
```
spring.data.mongodb.uri=${ENV_VAR} 
spring.data.mongodb.database=${ENV_VAR}
```

File: src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java (hunk 1 above)
Comment (thuyvy3110) [reply src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java]: Removed

File: src/main/java/com/dawn/jat/illuminati/comment/entity/CommentEntity.java (hunk 2 above)
Comment (thuyvy3110) [reply src/main/java/com/dawn/jat/illuminati/comment/entity/CommentEntity.java]: Removed

File: src/main/java/com/dawn/jat/illuminati/comment/repository/CommentRepository.java (hunk 3 above)
Comment (thuyvy3110) [reply src/main/java/com/dawn/jat/illuminati/comment/repository/CommentRepository.java]: Removed

File: src/main/java/com/dawn/jat/illuminati/comment/service/CommentService.java (hunk 4 above)
Comment (thuyvy3110) [reply src/main/java/com/dawn/jat/illuminati/comment/service/CommentService.java]: Removed

File: src/main/resources/application.properties (hunk 5 above)
Comment (thuyvy3110) [reply src/main/resources/application.properties]: Updated

File: src/main/resources/application-staging.properties (hunk 6 above)
Comment (thuyvy3110) [reply src/main/resources/application-staging.properties]: Updated

File: src/main/resources/application-prod.properties (hunk 7 above)
Comment (thuyvy3110) [reply src/main/resources/application-prod.properties]: Updated

Comment (thuyvy3110) [reply src/main/resources/application-prod.properties]: Updated

File: src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java (hunk 1 above)
Comment (thuyvy3110) [reply src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java]: Removed

**Pull request merged**